src/vs/base/browser/ui/list/list.ts

```diff
@@ -108,9 +120,9 @@ export interface IListDragAndDrop<T> extends IDisposable {
 	getDragURI(element: T): string | null;
 	getDragLabel?(elements: T[], originalEvent: DragEvent): string | undefined;
 	onDragStart?(data: IDragAndDropData, originalEvent: DragEvent): void;
-	onDragOver(data: IDragAndDropData, targetElement: T | undefined, targetIndex: number | undefined, originalEvent: DragEvent): boolean | IListDragOverReaction;
+	onDragOver(data: IDragAndDropData, targetElement: T | undefined, targetIndex: number | undefined, originalEvent: DragEvent, itemDndSector: ListViewItemDragAndDropSector | undefined): boolean | IListDragOverReaction;
```

Comment:
Should `itemDndSection` instead be `targetSector`?

Reply:
Should it also be placed between `targetIndex` and `originalEvent`, so all `target*` parameters are together?